client/src/components/completeProfile/index.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import styled from "styled-components";
+
+
+const CompleteProfile = () => {
+    return(
+      <BoxWrapper>
+          <Emoji><span role="img" aria-label="tada">✏️</span></Emoji>
+          <Text>Don't forget to complete your profile!</Text>
+      </BoxWrapper>
+    );
+};
+
+const BoxWrapper = styled.div`
+    margin: 2rem 4rem 0rem 4rem;
+    padding: 0.625rem;
+    width: auto;
+    background-color: white;
+    border-radius: 0.625rem;
+    display: flex;
+    box-shadow: 0px 0px 15px 0px #C1C1C1;
+    align-items: center;
+
+    @media (max-width: 768px) {
+        margin: 0rem 1rem 0rem 1rem;
```

Comment:
On mobile the two banners \~touch\~. Add top/bottom margins

```suggestion
        margin: 1rem;
```